### DISCUSSION TITLE
Support for COLBERT models in RAG

### DISCUSSION THREAD

**Discussion by rjmalagon** [open-webui/open-webui#3565] [General]
COLBERT models are an alternative to rerank, more complex but less resources needed, and with the correct implementation, maybe higher quality.

As I understand it, has more features than rerank models and needs more consideration to use.

https://huggingface.co/mixedbread-ai/mxbai-colbert-large-v1

I'm already trying to use colbert models similar to rerank models on open-webui. It seems to work, but I have doubt that it is correct to replace rerank with colbert directly.

Perhaps, possibly I am wrong about this, and colbert can be used just like a reranker, but I bring this as a feature request just in case.

**Comment by tjbck** [comment]
Currently investigating ColBERT, Keep us updated with  your findings if you manage to figure out the best way to integrate the model to our webui!
